### DISCUSSION TITLE
Issue with Automatic Library Export Feature Post-Recent Update on Mac M1

### DISCUSSION THREAD

**Discussion by olek233** [retorquere/zotero-better-bibtex#2703] [Q&A] [answered]
Hello,

First and foremost, I would like to extend my heartfelt appreciation for your dedication and hard work on the Better BibTeX plugin for Zotero. Your plugin has become an indispensable part of my daily academic routine, and I use it every day with great satisfaction.

However, after the recent update, I've encountered an issue regarding the automatic library export feature on my Mac M1. Specifically, the option to export the library to an external file automatically seems to have disappeared. I've attached a screenshot below to provide a clearer picture of what I am experiencing.

<img width="676" alt="Zrzut ekranu 2023-11-3 o 12 07 07" src="https://github.com/retorquere/zotero-better-bibtex/assets/22561923/6c1a8555-8f28-4066-a0e8-bf8785f686a5">

I would be grateful if you could provide some guidance on how I might resolve this issue to restore the automatic export functionality. Your help in this regard would be immensely appreciated, and I am more than willing to provide any additional information or carry out troubleshooting steps under your guidance.

Thank you very much for your time and assistance.

Best regards,
Alex

**Comment by retorquere** [comment]
I don't know what may have gone wrong without a debug log.

**Reply by retorquere** [reply]
Has nothing to do with M1.

**Reply by olek233** [reply]
It is here. 
[debug.txt](https://github.com/retorquere/zotero-better-bibtex/files/13320697/debug.txt)

**Reply by retorquere** [reply]
You don't have any auto exports set up.

**Reply by olek233** [reply]
How can I fix it?

**Reply by retorquere** [reply]
Check the checkbox "keep updated" when exporting.

**Reply by retorquere** [reply, marked as answer]
https://retorque.re/zotero-better-bibtex/exporting/auto/index.html

**Reply by olek233** [reply]
Thank you for your help. The problem was my fault - I did not understand that due to the update I should export the file again.

**Comment by cpraveen** [comment]
I may have same issue.

When I export in BibTeX format, I do not get a "Keep updated" option in the GUI. And then it does not show up under "Automatic export" tab also.

If I select "Better BibLaTeX" format, it does show "Keep updated" option.

Is the "Keep updated" option not available for exporting to BibTeX format ? It used to work in older versions.

**Reply by retorquere** [reply]
Not for BibTeX, is available for "Better BibTeX". If that doesn't show for you, I need a debug log.
